exporters/tokens/src/generators/fileGenerator.ts

```diff
@@ -182,4 +184,4 @@ export const generateOutputFilesByThemes = async (
   );
 
   return outputFiles;
-};
+};
```

Comment:
**nitpick (if-minor):** You are removing an empty line at the end of the file, I think. Can you fix this, please?

Reply:
empty line added back, thank you